kubernetes/replicator-gke-cc/docs/index.rst

```diff
@@ -0,0 +1,198 @@
+.. _quickstart-demos-operator-replicator-gke-cc:
+
+Google Kubernetes Engine to |ccloud| with |crep-full|
+=====================================================
+
+Overview
+--------
+
+This demonstrations features a deployment of |cp| on Google Kubernetes Engine (GKE) leveraging |co-long| and |crep-full|, highlighting a data replication strategy to |ccloud|.  Upon running this demo, you will have a GKE based |cp| deployment with simulated data replicating to your |ccloud| cluster.  We will verify the replication by running client applications against the |ccloud| cluster to view the simulated data originating in the source GKE cluster.
+
+The major components of this demo are:
+
+* A |ccloud| environment
```

Comment:
Technically speaking, there is something called an `environment` in CCloud.  So maybe better to just say cluster?

Reply:
Yeah, I intended to use the term environment here to mean "ccloud environment".

Reply:
I'll add the term cluster to the components list here